core/src/main/java/org/apache/iceberg/avro/AvroSchemaUtil.java

```diff
@@ -196,15 +199,38 @@ static Schema createProjectionMap(String recordName,
     return LogicalMap.get().addToSchema(Schema.createArray(keyValueRecord));
   }
 
-  private static int getId(Schema schema, String propertyName) {
+
+  static Integer getId(Schema schema, String propertyName) {
```

Comment:
I don't think there is a reason to remove the private modifier. It is still only used by other methods in this file.